app/src/main/res/values/strings.xml

```diff
@@ -16,5 +16,11 @@
 	<string name="fragment_request_search">Search</string>
     <string name="fragment_request_post_id">Post id</string>
     <string name="fragment_request_invalid_input">Invalid input, please enter a valid id.</string>
+    <string name="login_terms_and_conditions_text">By joining you agree to our</string>
+    <string name="login_terms_and_conditions_link"><u>Terms &amp; Conditions</u></string>
+    <string name="fragment_login_firstname">First Name (recommended)</string>
+    <string name="fragment_login_lastname">Last Name (recommended)</string>
+    <string name="fragment_login_button_log_in">Log In</string>
+    <string name="fragment_login_button_sign_up">Sign Up</string>
```

Comment:
A good practice is to "organize" this resources under a comment that works as a "Tag". Then all the resources will begin as the tag. It looks like this:
```
    <!-- Login -->
    <string name="login_terms_and_conditions_text">By joining you agree to our</string>
    <string name="login_terms_and_conditions_link"><u>Terms &amp; Conditions</u></string>
    <string name="login_firstname">First Name (recommended)</string>
    <string name="login_lastname">Last Name (recommended)</string>
    <string name="login_main_button">Log In</string>
    <string name="login_sign_up_button">Sign Up</string>
```

Reply:
It is not necessary to have the `fragment` prefix